README.md

```diff
@@ -141,11 +143,8 @@ We recommend querying using payment key hashes (`addr_vkh`) when possible (other
     // byron addresses, bech32 address, bech32 stake addresses or addr_vkh
     addresses: Array<string>,
     untilBlockHash: string, // only transactions up to this block (including it) will be considered for generating the diff
-    afterPoint?: {
-      blockHash: string, // only transactions AFTER this clock will be considered for generating the diff
-      itemIndex?: number // if `itemIndex` is supplied, we will also consider transactions from `blockHash`, but only the outputs which have a bigger index than `itemIndex`
-    },
-    afterBestBlocks?: Array<string> // only transactions after the latest block from this array will be included. The inclusion of `afterBestBlocks` in the request will also add 2 new fields to the response (see the response bellow for more details)
```

Comment:
@yushih , I agree that we can get rid of the `afterPoint` query format for now at all, only use querying by full blocks and then add anything else later, if needed.

**BUT** in this case we only need to completely remove the `afterPoint` parameter and **preserve** the `afterBestBlocks` parameter in the form as it's described in the documentation.

The `afterBestBlocks` parameter exists as an array for a very specific reason, because the query protocol is based on knowing both the bestblock and the safeblock and this information must be atomic with the querying itself and cannot be checked separately.

 If the `afterBestBlocks` case have been already implemented before this change, there should be no problem to just get rid of the `afterPoint` case. The only other change that would be needed is to make sure the query returns items by full blocks without limiting items in a middle of a block